### PR TITLE
VG-4244 - saved searches dont include everything

### PR DESCRIPTION
**Pull request by john-atkinson-oci**

### CODE REVIEW

**Comment by ryantxu** [comment]
Can one of the admins verify this patch?

File: app/src/saved-search/saved-search-service.js [hunk 1]
```diff
@@ -247,6 +247,30 @@ angular.module('voyager.search').
...
                 return sugar.postForm('api/rest/display/ssearch/' + id + '/order', data);
+            },
+            sortSavedSearches: function(savedSearches) {
+                var global = [], personal = [];
+                var userHasSaveSearch = authService.hasPermission('save_search');
+                var authUser = authService.getUser();
+                $.each(savedSearches, function(index, saved) {
+                    if((authUser) && (userHasSaveSearch))
+                    {
+                        if(saved.owner === authService.getUser().name) {
```
Comment (scottlepp) [inline app/src/saved-search/saved-search-service.js]: @john-atkinson-oci looks like you can just use the authUser variable here?

File: app/src/saved-search/saved-searches.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div ng-controller="SavedSearchCtrl">
+<div ng-controller="SavedSearchCtrl as savedSearch">
```
Comment (scottlepp) [inline app/src/saved-search/saved-searches.html]: Rename "savedSearch" to "vm" to be consistent with the vm variable in the controller. This is the style guide standard too.

File: app/src/home/home.html [hunk 3]
```diff
@@ -61,21 +61,21 @@
 						</li>
 					</ul>
 				</article>
-				<article ng-hide="showRecent" ng-controller="HomeSavedSearchCtrl">
-					<ul ng-show="personalSavedSearches.length">
-						<li ng-repeat="search in personalSavedSearches">
-							<a href="javascript:;" ng-if="hasPermission('save_search')" ng-click="deleteSearch(search.id)" class="favorite_link">
+				<article ng-hide="showRecent" ng-controller="SavedSearchCtrl as homeSavedSearch">
```
Comment (scottlepp) [inline app/src/home/home.html]: The standard practice we're using when using controller as is to call it "vm".  This follows our style guide.

File: app/src/saved-search/home-saved-search-controller.js [hunk 4]
```diff
@@ -1,57 +0,0 @@
-//TODO duplication with SavedSearchCtrl - add base class
```
Comment (scottlepp) [inline app/src/saved-search/home-saved-search-controller.js]: Nice!  Thanks for removing this redundant code.

**Pull request merged**